extensions/mapstore/src/main/java/com/hazelcast/mapstore/GenericMapStoreProperties.java

```diff
@@ -61,6 +67,10 @@ class GenericMapStoreProperties {
 
         idColumnInColumns = columns.isEmpty() || columns.contains(idColumn);
         compactTypeName = properties.getProperty(TYPE_NAME_PROPERTY, mapName);
+
+        String value = properties.getProperty(LOAD_ALL_KEYS_PROPERTY, "true");
```

Comment:
Wouldn't it be better to fail in this case? If the user accidentally writes `True` it will silently load nothing?

Reply:
rev: 406fc8e520aaa338af760704b54aab627c851373

Reply:
Still different behaviour than here https://github.com/hazelcast/hazelcast/pull/24331/files#diff-dc9f331cfdacf836dd9470a6095f88b55e1e2135be51932276622e358cc68e18R187

Reply:
I think it is ok, in `validateMapStoreConfig` we are stricter - we accept null, `true` and `false`.
Null defaults to `true`. The `parseBoolean` accepts more, but both validated values are parsed as expected.

Or do you still see some issue?